modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/UserCardTag.java

```diff
@@ -40,186 +38,73 @@ public int doStartTag() throws JspException {
 		return super.doStartTag();
 	}
 
-	public List<DropdownItem> getActionDropdownItems() {
-		if ((_actionDropdownItems == null) && (_userCard != null)) {
-			return _userCard.getActionDropdownItems();
-		}
-
-		return _actionDropdownItems;
-	}
-
 	@Override
-	public String getCssClass() {
-		if ((super.getCssClass() == null) && (_userCard != null)) {
-			if (_userCard.getCssClass() != null) {
-				return _userCard.getCssClass();
-			}
-
-			if (_userCard.getElementClasses() != null) {
-				return _userCard.getElementClasses();
-			}
-		}
-
-		return super.getCssClass();
-	}
-
-	public String getHref() {
-		if ((_href == null) && (_userCard != null)) {
-			return _userCard.getHref();
-		}
-
-		return _href;
-	}
-
 	public String getIcon() {
-		if (_icon == null) {
-			if (_userCard != null) {
-				return _userCard.getIcon();
+		String icon = super.getIcon();
+
+		if (icon == null) {
+			if ((getUserCard() != null) && (getUserCard().getIcon() != null)) {
+				return getUserCard().getIcon();
```

Comment:
I don't think that Brian allows chaining in backend :slightly_smiling_face:

Reply:
Updated ;)